hack/release.sh

```diff
@@ -133,4 +133,6 @@ for yaml in "${RELEASE_YAML}" "${LITE_YAML}" "${NO_MON_YAML}" "${SERVING_YAML}"
   publish_yaml "${yaml}" "${SERVING_RELEASE_GCS}" "${TAG}"
 done
 
+branch_release "Knative Serving" "${RELEASE_YAML} ${LITE_YAML} ${NO_MON_YAML}"
```

Comment:
At a minimum, we should also publish `serving.yaml`, `build.yaml`, `monitoring.yaml`, but I think we should include everything in the list published above.

Reply:
Done.